decidim-core/app/packs/src/decidim/input_character_counter.js

```diff
@@ -91,10 +91,16 @@ export default class InputCharacterCounter {
 
     if (this.maxCharacters > 0) {
       const remaining = this.maxCharacters - numCharacters;
+      let remainingCharacters = this.$input.data("remaining-characters")
       let message = MESSAGES.charactersLeft.other;
       if (remaining === 1) {
         message = MESSAGES.charactersLeft.one;
       }
+      if (document.querySelector(`textarea[data-remaining-characters="${remainingCharacters}"]`)) {
```

Comment:
I think this if statement is unnecessary, isn't it?

Could you please remove it?